package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-chat-window",
-  "version": "1.1.0",
+  "version": "1.2.1",
```

Comment:
Which changes are we counting for this release?

Reply:
Hmm I don't know why this would change, readme changes shouldn't count... I'll undo this